src/easylink/step.py

```diff
@@ -1002,8 +1002,13 @@ def _duplicate_template_step(self) -> Step:
 class LoopStep(TemplatedStep):
     """A type of :class:`TemplatedStep` that allows for looping.
 
-    A ``LoopStep`` allows a user to loop a single :class:`Step` or a sequence
-    of ``Steps`` multiple times such that each iteration depends on the previous.
+    A ``LoopStep`` allows a user to loop a :class:`Step` (or a sequence
+    of ``Steps`` in the case of a :class:`HierarchicalStep`) multiple times such that 
+    each iteration depends on the previous.
+
+    In the pipeline configuration an implementation must be specified for each 
+    iteration of the step.
```

Comment:
This seems not quite right, because it might not be a single implementation if you looped an entire section. Maybe:
```suggestion
    In the pipeline configuration the implementation(s) must be specified for each 
    iteration of the looped step.
```